web/src/app/admin/admin-alert-counts/AlertCountLineGraph.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import { Grid, Paper, Typography } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles/makeStyles'
+import { Theme, useTheme } from '@mui/material/styles'
+import {
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  ResponsiveContainer,
+  LineChart,
+  Line,
+  Legend,
+  Tooltip,
+} from 'recharts'
+
+interface AlertCountLineGraphProps {
+  data: typeof LineChart.defaultProps['data']
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  graphContent: {
+    height: '500px',
+    fontFamily: theme.typography.body2.fontFamily,
+  },
+  bar: {
+    '&:hover': {
+      cursor: 'pointer',
+    },
+  },
+}))
+
+export default function AlertCountLineGraph(
+  props: AlertCountLineGraphProps,
+): JSX.Element {
+  const [active, setActive] = useState('')
+  const classes = useStyles()
+  const theme = useTheme()
+
+  const chooseColor = (idx: number): string => {
+    switch (idx) {
+      case 1:
+        return '#3f8e98'
+      case 2:
+        return '#209e90'
+      case 3:
+        return '#41ab74'
+      case 4:
+        return '#79b34a'
+      case 5:
+        return '#b9b218'
+      case 6:
+        return '#ffa600'
+      default:
+        return '#607d8b'
+    }
+  }
```

Comment:
thoughts on something like this?

```
import {
    blueGrey,
    teal,
    green,
    cyan,
    amber,
    pink,
    brown,
} from '@mui/material/colors'
```

```suggestion
  const chooseColor = (idx: number): string => {
    const shade = theme.mode.light ? 'A700' : 'A400'
  
    switch (idx) {
      case 1:
        return teal[shade]
      case 2:
        return brown[shade]
      case 3:
        return green[shade]
      case 4:
        return cyan[shade]
      case 5:
        return amber[shade]
      case 6:
        return pink[shade]
      default:
        return blueGrey[shade]
    }
  }
```

Reply:
Yea I like these colors, feels more in line with the rest of our color theming too 👍🏽